plugins/polio/models.py

```diff
@@ -371,27 +371,29 @@ class Meta:
     budget_current_state_label = models.CharField(max_length=100, null=True, blank=True)
 
     # Budget tab
-    budget_requested_at_WFEDITABLE = models.DateField(null=True, blank=True)
     who_sent_budget_at_WFEDITABLE = models.DateField(null=True, blank=True)
     unicef_sent_budget_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    gpei_consolidation_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    gpei_consolidated_budgets_at_WFEDITABLE = models.DateField(null=True, blank=True)
     submitted_to_rrt_at_WFEDITABLE = models.DateField(null=True, blank=True)
     feedback_sent_to_gpei_at_WFEDITABLE = models.DateField(null=True, blank=True)
     re_submitted_to_rrt_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    submission_to_orpg_operations_1_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    submitted_to_orpg_operations1_at_WFEDITABLE = models.DateField(null=True, blank=True)
     feedback_sent_to_rrt1_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    submitted_to_orpg_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    submitted_to_orpg_wider_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    submitted_to_orpg_operations2_at_WFEDITABLE = models.DateField(null=True, blank=True)
     feedback_sent_to_rrt2_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    re_submitted_to_orpg_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    submission_to_orpg_operations_2_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    feedback_sent_to_rrt3_at_WFEDITABLE = models.DateField(null=True, blank=True)
-    re_submission_to_orpg_operations_2_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    re_submitted_to_orpg_operations2_at_WFEDITABLE = models.DateField(null=True, blank=True)
     submitted_for_approval_at_WFEDITABLE = models.DateField(null=True, blank=True)
     feedback_sent_to_orpg_operations_unicef_at_WFEDITABLE = models.DateField(null=True, blank=True)
     feedback_sent_to_orpg_operations_who_at_WFEDITABLE = models.DateField(null=True, blank=True)
     approved_by_who_at_WFEDITABLE = models.DateField(null=True, blank=True)
     approved_by_unicef_at_WFEDITABLE = models.DateField(null=True, blank=True)
     approved_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    approval_confirmed_at_WFEDITABLE = models.DateField(null=True, blank=True)
+    # LEGACY deprecated fields
+    budget_requested_at_WFEDITABLE_old = models.DateField(null=True, blank=True)
+    feedback_sent_to_rrt3_at_WFEDITABLE_old = models.DateField(null=True, blank=True)
+    re_submitted_to_orpg_at_WFEDITABLE_old = models.DateField(null=True, blank=True)
```

Comment:
I still see the old field in BudgetForm.js and useFormValidator.js is that normal?

![image](https://user-images.githubusercontent.com/82500/225640044-fca15671-c7d5-4659-a71e-2aad34692a67.png)